### PR TITLE
Added dRPC to the API list.

### PR DESCRIPTION
**Pull request by maradeeym**
## Summary by CodeRabbit

- **New Features**
	- Updated the documentation to include "dRPC" as a recommended RPC partner, replacing "Blast API," enhancing guidance for developers seeking private endpoints.

### CODE REVIEW

**Comment by fadeev** [comment]
@maradeeym it's great to see dRPC supporting ZetaChain! Let's discuss adding dRPC to the list of recommended providers, feel free to ping me at https://t.me/dfadeev

File: src/pages/reference/network/api.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { EndpointList } from "~/components/Docs";
 If you are a developer we highly recommend signing up for a free private
 endpoint through [AllThatNode](https://www.allthatnode.com/zetachain.dsrv),
 [BlockPI](https://blockpi.io/zeta), [InfStones](https://infstones.com/fast-api),
-[Blast API](https://blastapi.io/chains/zetachain) or one of our other RPC
+[Blast API](https://blastapi.io/chains/zetachain), [dRPC](https://drpc.org/chainlist/zeta-chain) or one of our other RPC
```
Comment (CharlieMc0) [inline src/pages/reference/network/api.mdx:12]: Let's change it to `, [dRPC](https://drpc.org/chainlist/zeta-chain), or another RPC
provider.`

File: src/pages/reference/network/api.mdx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { EndpointList } from "~/components/Docs";
...
 partners. The private endpoints have higher throughput, less restrictions, and
```
Comment (CharlieMc0) [inline src/pages/reference/network/api.mdx:13]: Let's change it to , [dRPC](https://drpc.org/chainlist/zeta-chain), or another RPC provider.

Comment (fadeev) [reply src/pages/reference/network/api.mdx:13]: @maradeeym after this change we'll be ready to merge 👍

**Review by andresaiello — APPROVED** [review]

File: src/pages/reference/network/api.mdx (hunk 2 above)
Comment (fadeev) [reply src/pages/reference/network/api.mdx:13]: I'll update this in a separate PR.

**Pull request merged**